### ISSUE TITLE
Kolobok has wrong behaviour specifically in SRP  

### ISSUE THREAD

**Issue by SurDno** [Decane/SRP#54]
In vanilla game, if you attack Duty Base, Kolobok just stands with his hands up.

In SRP, however, he starts running around his bar like any other stalker with no weapon.
![20181104004959_1](https://user-images.githubusercontent.com/35204405/48676500-df508880-eb78-11e8-9ce5-dc1888b38cea.jpg)

**Comment by Marek33** [comment]
He should get a weapon, and then it's fine, even if I don't think you should have an option to have the weapons out at the base. However, since there are Fraction wars, he should have a weapon and then it would be fixed.

**Comment by Decane** [comment]
His behavior is unmodified from vanilla. He flees whenever `!is_smart_in_combat(agr_smart_terrain_1_6)` and he has detected a danger source - most frequently the player. I accept that his behavior is erratic and should be improved.

**Comment by SurDno** [comment]
> He should get a weapon, and then it's fine, even if I don't think you should have an option to have the weapons out at the base. However, since there are Fraction wars, he should have a weapon and then it would be fixed.

You do not seem to understand. When you attack the base, he stands still even **WITHOUT** a weapon, just with his hands up. In vanilla the same applies to two more people on the same location: Thrush and Aydar. SRP also implements that behaviour for Lambee.

![image](https://user-images.githubusercontent.com/35204405/48679506-37e84b80-eba2-11e8-9450-8f5f631c32db.png)

However, as far as I understand, this behaviour does not activate when Duty's (or Stalkers') Base is no longer attacked. I suggest replacing this check to just checking if the weapon is pointed on these guys, because further things happen with vanilla implementation:

1. If you have slugged the whole Stalker's Base but important characters (then game thinks the point is no longer attacked), **Thrush** will start doing all of his animations with a gun. If you start pointing your gun on him, he'll do the same no matter if you're neutral or enemy with him.

![image](https://user-images.githubusercontent.com/35204405/48679875-6f58f700-eba6-11e8-8d59-37b636b1d909.png)

2. If you have slugged the whole Stalker's Base but important characters (then game thinks the point is no longer attacked), if you start point your gun on **Aydar**, he'll just stand in.... this pose, probably because he has no gun. He'll do it no matter if you're neutral or enemy with him.

![image](https://user-images.githubusercontent.com/35204405/48679852-14270480-eba6-11e8-8feb-a8c036ab99f2.png)

3. If you slugged the whole Duty Base but important characters (then game thinks the point is no longer attacked), if you start point your gun on **Kolobok**, he won't react if you're neutral (you may interact with him as if you haven't attacked Duty Base. _However_, if you're hostile to Kolobok, he'll start fleeing instead of surrendering.

![image](https://user-images.githubusercontent.com/35204405/48679904-ba730a00-eba6-11e8-8776-8e8d87e79015.png)


If their hands up behaviour **is intended** by GSC, but does not activate in some cases, I think it's more of an oversight (probably to prevent this from happening when player is _not_ attacking Duty's/Stalkers' Base), and I doubt that fixing this would be enhancement.

@Decane, what do you think?

**Comment by Marek33** [comment]
Yes, I know that. However, I never go hostile against any fraction, since I can easily have a lot of money by selling artefacts. Also, I have never seen that behavior expects that one soldier in the Cordon because of that.
